probe-rs-targets/targets/nRF52832.yaml

```diff
@@ -1,9 +1,8 @@
 name: "nRF52832"
-# TODO: Manufacturer and Part are not correct yet. They are only needed for autodetection.
 manufacturer:
   cc: 0x02
   id: 0x44
-part: 0x000111
+part: 0x000006
```

Comment:
This fixes parts of #52 right? Can you put it in a separate commit / PR which we can get merged faster?

Reply:
Yeah, totally forgot about that. Now it's in a separate commit